src/main/kotlin/no/digdir/informasjonsforvaltning/fdk_mqa_dcat_validator/configuration/ApplicationProperties.kt

```diff
@@ -0,0 +1,15 @@
+package no.digdir.informasjonsforvaltning.fdk_mqa_dcat_validator.configuration
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.context.annotation.Bean
+
+@ConfigurationProperties("application")
+data class ApplicationProperties(
+    val prop: String
+)
+
+@Bean
+fun objectMapper(): ObjectMapper =
+    ObjectMapper().enable(DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_USING_DEFAULT_VALUE)
```

Comment:
enum?

Reply:
Når enum som kommer fra kafka ikke eksisterer at den da settes til default istedenfor enum not found exception

Reply:
ok, ja. Tenkte det skulle ha noe med `ApplicationProperties` du definerer ovenfor, men du la bare litt konfigurering av ObjectMapper i samme fil. Carry on :+1: